fml/gpu_thread_merger.cc

```diff
@@ -63,6 +63,7 @@ GpuThreadStatus GpuThreadMerger::DecrementLease() {
   lease_term_--;
   if (lease_term_ == 0) {
     bool success = task_queues_->Unmerge(platform_queue_id_);
+    FML_CHECK(success) << "Unable to un-merge the GPU and platform threads.";
     is_merged_ = !success;
```

Comment:
nit: this can now be `is_merged_ = false`

Reply:
DONE!